llvm/test/CodeGen/AMDGPU/fix-illegal-vgpr-copies.mir

```diff
@@ -0,0 +1,29 @@
+# RUN: not --crash llc -march=amdgcn -mcpu=gfx1200 -run-pass=machineverifier %s 2>&1 | FileCheck -check-prefix=ERR %s
```

Comment:
This belongs in test/MachineVerifier. Also you can just use -run-pass=none, the verifier runs on the input anyway

Reply:
Done.